src/utils/formatting/content_split.rs

```diff
@@ -28,8 +28,8 @@ pub fn split_line(line: &str, info: &ColumnDisplayInfo, delimiter: char) -> Vec<
 
```

Comment:
The current logic allows splits on arbitrary characters, including characters that could be contained inside of ansi escape codes.
We would have to somehow ensure that this won't break